core/source/String.mint

```diff
@@ -213,4 +213,44 @@ module String {
   fun fromArray (array : Array(String)) : String {
     concat(array)
   }
+
+  /*
+  Returns whether or not the string is not empty (only contains whitespace).
+
+    String.isNotEmpty("a") == true
+    String.isNotEmpty("   ") == false
+  */
+  fun isNotEmpty (string : String) : Bool {
+    !String.isEmpty(String.trim(string))
```

Comment:
IMO that should be 1:1 reversible with `String.isEmpty`:

```suggestion
    !String.isEmpty(string)
```

Desired behaviour should be a part of `String.isBlank` instead.

Reply:
@gdotdesign Still unresolved.